code/modules/lighting/lighting_object.dm

```diff
@@ -40,7 +39,6 @@
 			stack_trace("A lighting object was qdeleted with a different loc then it is suppose to have ([COORD(oldturf)] -> [COORD(newturf)])")
 		if (isturf(myturf))
 			myturf.lighting_object = null
-			myturf.luminosity = initial(myturf.luminosity)
```

Comment:
These lines of code where unnecessary because lighting objects actually update luminosity of themselves, rather than the luminosity of the turfs that they affect. This is an artifact left over from overlay lighting and may incorrectly break future turfs that:
- Use some dynamic lum sources
- Have their area's lighting removed for some reason